receiver/sqlserverreceiver/testdata/expectedQueryTextAndPlanQueryWithInvalidData.yaml

```diff
@@ -18,7 +18,7 @@ resourceLogs:
                   doubleValue: 0
               - key: db.query.text
                 value:
-                  stringValue: SELECT cpu_time AS [CPU Usage (time)]
+                  stringValue: SELECT cpu_time AS [CPU Usage (time)
```

Comment:
Can you share more context as to why the matching end bracket is being dropped for this obfuscation change?